bin/reference_data.py

```diff
@@ -11,6 +12,8 @@
 from egcg_core.app_logging import AppLogger, logging_default
 from egcg_core.config import cfg
 from egcg_core.exceptions import EGCGError
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
That's  a small bugfix so we don't need to set the PYTHON PATH when running `reference_data.py`